src/locales/en-US/framework.ftl

```diff
@@ -3,3 +3,38 @@
 ### among different targets.
 
 framework-validation-required = This field is required.
+
+framework-timeago =
+  { $value ->
```

Comment:
The `Spanish` translations is using `en 1 hora` but the English translation `1 hour from now`, we should stay consistent and also translate to `in 1 hour`.

Reply:
💅 : ok

Reply:
I tried that the last time actually and I first didn't know how to do it, but now I do, we can talk about it on Monday too :-)

Reply:
Actually in our current Talk we also changed from `1 hour from now` to `in 1 hour`, I don't know any reason who would use `1 hour from now`.